webpack/assets/javascripts/react_app/components/LoginPage/LoginPage.js

```diff
@@ -24,10 +23,7 @@ const LoginPage = ({
         header={{
           logoSrc,
           caption: (
-            <>
               <h1 id="title">{__('Welcome')}</h1>
-              {version && <p id="version">{`${__('Version')} ${version}`}</p>}
```

Comment:
instead of removing it, I would suggest adding a new setting that sets whether to display the setting or not, and then in https://github.com/theforeman/foreman/blob/develop/app/helpers/login_helper.rb#L5 we can pass it or not based on that setting.

Reply:
Apologies, there was a long discussion on the security list about how to approach this and folks did not like a new setting. So we decided either to remove the version completely, or move the text into the footer text which was a bit more flexible since QA do require version to be present for some tests.

Reply:
Oh I see, it's still feels a bit strange to display `version $VERSION` in case the admins didn't replace the variable.
Adding @marisvirik for another opinion,
Maybe we can test if the $VERSION var has changed and only if it did, we should display it, does it makes sense?

Reply:
You need to move the i18n of the "Version" word from JS here into Ruby. Problem is that we initialize default value during seed I think, if user changes language it will not be translated. I think we need to live with that.